Content.Server/AI/Operators/Inventory/DropEntityOperator.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Server.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+
+namespace Content.Server.AI.Operators.Inventory
+{
+    public class DropEntityOperator : AiOperator
+    {
+        private readonly IEntity _owner;
+        private readonly IEntity _entity;
+        public DropEntityOperator(IEntity owner, IEntity entity)
+        {
+            _owner = owner;
+            _entity = entity;
+        }
+
+        /// <summary>
+        /// Requires EquipEntityOperator to put it in the active hand first
+        /// </summary>
+        /// <param name="frameTime"></param>
+        /// <returns></returns>
+        public override Outcome Execute(float frameTime)
+        {
+            if (!_owner.TryGetComponent(out HandsComponent handsComponent))
+            {
+                return Outcome.Failed;
+            }
+
+            foreach (var hand in handsComponent.ActivePriorityEnumerable())
+            {
+                if (handsComponent.GetHand(hand).Owner == _entity)
+                {
+                    if (handsComponent.CanDrop(hand))
+                    {
+                        handsComponent.Drop(_entity);
+                        return Outcome.Success;
+                    }
+
+                    return Outcome.Failed;
+                }
+            }
```

Comment:
As far as I can tell, this could be replaced with
```csharp
if (handsComponent.Drop(_entity))
{
    return Outcome.Success;
}
return Outcome.Failed;
```
if `_entity` is guaranteed to be in one of the hands - but I don't see why that wouldn't be the case.

Reply:
If the utilityaction using it doesn't have the right considerations it could be getting called when it doesn't actually hold the item. I've added a log warning to it as it probably shouldn't happen.

I probably brain-farted on the drop.